src/main/java/net/snowflake/ingest/utils/IcebergDataTypeParser.java

```diff
@@ -208,4 +223,61 @@ public static Types.MapType mapFromJson(JsonNode json) {
       return Types.MapType.ofOptional(keyId, valueId, keyType, valueType);
     }
   }
+
+  private static org.apache.parquet.schema.Type replaceWithOriginalFieldName(
```

Comment:
add method comment on why we're having to do this?

Reply:
Added